src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/checkout_services/adyenCheckoutServices.js

```diff
@@ -0,0 +1,42 @@
+const constants = require('*/cartridge/adyenConstants/constants');
+const collections = require('*/cartridge/scripts/util/collections');
+
+function processPayment(order, handlePaymentResult, req, res, emit) {
+  res.json({
+    error: false,
+    adyenAction: handlePaymentResult.action,
+    orderID: order.orderNo,
+    orderToken: order.orderToken,
+  });
+  emit('route:Complete');
+}
+
+function isAdyen(currentBasket, next) {
```

Comment:
this function return type is inconsistent, it a boolean, or `next`. Unless next is a boolean, we could consider removing it from this function to make it more consistent and do
```
if(isAdyen) {
  return next();
}
```
in CheckoutServices.js